### ISSUE TITLE
Error 500 when using JupyterHub REST API

### ISSUE THREAD

**Issue by cbjuan** [jupyterhub/jupyterhub#2491]
**Describe the bug**

I receive a 500 error when making requests on some JupyterHub REST API endpoints.
I get the error when using the endpoints `/hub/api/users/`, `/hub/api/user/username/server` at least. I don't know which other methods would return the same error.

Trace of the error:
```
[E 2019-03-29 14:26:11.172 JupyterHub web:1788] Uncaught exception GET /hub/api/users/ (10.177.140.55)
    HTTPServerRequest(protocol='http', host='ibm-q-jupyterhub-dev.us-south.containers.appdomain.cloud', method='GET', uri='/hub/api/users/', version='HTTP/1.1', remote_ip='xxxxxxxxx')
    Traceback (most recent call last):
      File "/home/jovyan/.local/lib/python3.6/site-packages/tornado/web.py", line 1678, in _execute
        result = await result
      File "/home/jovyan/.local/lib/python3.6/site-packages/jupyterhub/apihandlers/base.py", line 279, in prepare
        await super().prepare()
    TypeError: object NoneType can't be used in 'await' expression
    
[E 2019-03-29 14:26:11.194 JupyterHub log:150] {
      "Accept-Encoding": "gzip, deflate",
      "Accept": "*/*",
      "User-Agent": "PostmanRuntime/7.6.1",
      "Postman-Token": "xxxxx",
      "Cache-Control": "no-cache",
      "Authorization": "token [secret]",
      "X-Forwarded-Proto": "https,http",
      "X-Forwarded-Port": "443,80",
      "X-Forwarded-Host": "xxxxxx",
      "X-Forwarded-For": "xxxxxx,xxxxxx",
      "X-Real-Ip": "xxxxx",
      "Host": "xxxxxxx",
      "X-Global-K8fdic-Transaction-Id": "de5b5dcd318f9a975133c26272e52d38",
      "Connection": "close"
    }
[E 2019-03-29 14:26:11.194 JupyterHub log:158] 500 GET /hub/api/users/ (5afebbde93c45aeca8c4f73c@10.xxx.xxx.xx) 2.75ms
```
JupyterHub version: `0.9.5`
Tornado version: `6.0.2`

**To Reproduce**
Steps to reproduce the behavior: with the JupyterHub deployed, make a REST API like: 
```
curl -X GET \
  https://host/hub/api/users/ \
  -H 'Authorization: token tokenFromAnAdminUser' \
```
**Expected behavior**
Get the proper response from the REST endpoint intead of a 500 error.

**Additional context**
I have deployed the JupyterHub using the `k8s` version. Helm chart: `0.8.1`. Also it happens with version `0.8.0`

Output from`jupyter troubleshoot --debug` 
```
$PATH:
	/opt/conda/bin
	/opt/conda/bin
	/usr/local/sbin
	/usr/local/bin
	/usr/sbin
	/usr/bin
	/sbin
	/bin

sys.path:
	/opt/conda/bin
	/opt/conda/lib/python36.zip
	/opt/conda/lib/python3.6
	/opt/conda/lib/python3.6/lib-dynload
	/opt/conda/lib/python3.6/site-packages

sys.executable:
	/opt/conda/bin/python3.6

sys.version:
	3.6.7 | packaged by conda-forge | (default, Nov 21 2018, 02:32:25) 
	[GCC 4.8.2 20140120 (Red Hat 4.8.2-15)]

platform.platform():
	Linux-4.15.0-46-generic-x86_64-with-debian-buster-sid

which -a jupyter:
	/opt/conda/bin/jupyter
	/opt/conda/bin/jupyter

pip list:
	Package                           Version 
	--------------------------------- --------
	alembic                           0.9.9   
	asn1crypto                        0.24.0  
	async-generator                   1.10    
	attrs                             18.2.0  
	autopep8                          1.4.3   
	backcall                          0.1.0   
	beautifulsoup4                    4.6.3   
	bleach                            3.0.2   
	bokeh                             0.13.0  
	certifi                           2019.3.9
	cffi                              1.12.2  
	chardet                           3.0.4   
	cloudpickle                       0.5.6   
	conda                             4.5.12  
	cryptography                      2.6.1   
	cryptography-vectors              2.3.1   
	cvxopt                            1.2.3   
	cycler                            0.10.0  
	Cython                            0.28.5  
	cytoolz                           0.9.0.1 
	dask                              1.0.0   
	decorator                         4.4.0   
	dill                              0.2.8.2 
	entrypoints                       0.2.3   
	fastcache                         1.0.2   
	gmpy2                             2.0.8   
	h5py                              2.9.0   
	idna                              2.8     
	imageio                           2.4.1   
	ipykernel                         5.1.0   
	ipython                           7.2.0   
	ipython-genutils                  0.2.0   
	ipywidgets                        7.2.1   
	jedi                              0.13.2  
	Jinja2                            2.10    
	jsonschema                        3.0.0a3 
	jupyter-client                    5.2.4   
	jupyter-contrib-core              0.3.3   
	jupyter-contrib-nbextensions      0.5.1   
	jupyter-core                      4.4.0   
	jupyter-highlight-selected-word   0.2.0   
	jupyter-latex-envs                1.4.6   
	jupyter-nbextensions-configurator 0.4.1   
	jupyterhub                        0.9.4   
	jupyterlab                        0.35.4  
	jupyterlab-server                 0.2.0   
	kiwisolver                        1.0.1   
	llvmlite                          0.23.0  
	lxml                              4.3.3   
	Mako                              1.0.7   
	MarkupSafe                        1.1.0   
	marshmallow                       2.19.1  
	marshmallow-polyfield             3.2     
	matplotlib                        2.2.3   
	mistune                           0.8.4   
	mpmath                            1.1.0   
	nbconvert                         5.3.1   
	nbformat                          4.4.0   
	nbgitpuller                       0.6.1   
	networkx                          2.2     
	notebook                          5.7.2   
	ntlm-auth                         1.2.0   
	numba                             0.38.1  
	numexpr                           2.6.9   
	numpy                             1.16.2  
	olefile                           0.46    
	packaging                         18.0    
	pamela                            0.3.0   
	pandas                            0.23.4  
	pandocfilters                     1.4.2   
	parso                             0.3.1   
	patsy                             0.5.1   
	pexpect                           4.6.0   
	pickleshare                       0.7.5   
	Pillow                            5.4.1   
	pip                               18.1    
	ply                               3.11    
	prometheus-client                 0.5.0   
	prompt-toolkit                    2.0.7   
	protobuf                          3.6.1   
	psutil                            5.6.1   
	ptyprocess                        0.6.0   
	pycodestyle                       2.5.0   
	pycosat                           0.6.3   
	pycparser                         2.19    
	pycurl                            7.43.0.2
	Pygments                          2.3.1   
	pyOpenSSL                         18.0.0  
	pyparsing                         2.3.0   
	pyrsistent                        0.14.8  
	PySocks                           1.6.8   
	python-dateutil                   2.7.5   
	python-editor                     1.0.3   
	python-oauth2                     1.0.1   
	pytz                              2018.7  
	PyWavelets                        1.0.1   
	PyYAML                            3.13    
	pyzmq                             17.1.2  
	qiskit                            0.8.0   
	qiskit-aer                        0.1.1   
	qiskit-aqua                       0.4.1   
	qiskit-chemistry                  0.4.2   
	qiskit-ignis                      0.1.0   
	qiskit-terra                      0.7.1   
	requests                          2.21.0  
	requests-ntlm                     1.1.0   
	rpy2                              2.9.4   
	ruamel-yaml                       0.15.71 
	scikit-image                      0.14.1  
	scikit-learn                      0.20.2  
	scipy                             1.2.1   
	seaborn                           0.9.0   
	Send2Trash                        1.5.0   
	setuptools                        40.8.0  
	six                               1.12.0  
	SQLAlchemy                        1.2.15  
	statsmodels                       0.9.0   
	sympy                             1.3     
	terminado                         0.8.1   
	testpath                          0.4.2   
	toolz                             0.9.0   
	tornado                           5.1.1   
	traitlets                         4.3.2   
	tzlocal                           1.5.1   
	urllib3                           1.24.1  
	vincent                           0.4.4   
	wcwidth                           0.1.7   
	webencodings                      0.5.1   
	wheel                             0.32.3  
	widgetsnbextension                3.2.1   
	xlrd                              1.2.0

conda list:
	# packages in environment at /opt/conda:
	#
	# Name                    Version                   Build  Channel
	_r-mutex                  1.0.0               anacondar_1    defaults
	alembic                   0.9.9                      py_0    conda-forge
	asn1crypto                0.24.0                    <pip>
	asn1crypto                0.24.0                py36_1003    conda-forge
	async_generator           1.10                       py_0    conda-forge
	attrs                     18.2.0                     py_0    conda-forge
	autopep8                  1.4.3                     <pip>
	backcall                  0.1.0                      py_0    conda-forge
	beautifulsoup4            4.6.3                 py36_1000    conda-forge
	blas                      1.1                    openblas    conda-forge
	bleach                    3.0.2                      py_1    conda-forge
	bokeh                     0.13.0                py36_1000    conda-forge
	bwidget                   1.9.11                        1    defaults
	bzip2                     1.0.6                h470a237_2    conda-forge
	ca-certificates           2018.11.29           ha4d7672_0    conda-forge
	cairo                     1.14.12              h276e583_5    conda-forge
	certifi                   2018.11.29            py36_1000    conda-forge
	certifi                   2019.3.9                  <pip>
	cffi                      1.12.2                    <pip>
	cffi                      1.11.5           py36h5e8e0c9_1    conda-forge
	chardet                   3.0.4                     <pip>
	chardet                   3.0.4                 py36_1003    conda-forge
	cloudpickle               0.5.6                      py_0    conda-forge
	conda                     4.5.12                py36_1000    conda-forge
	conda-env                 2.6.0                         1    conda-forge
	configurable-http-proxy   3.1.0                   node8_1    conda-forge
	cryptography              2.3.1            py36hdffb7b8_0    conda-forge
	cryptography              2.6.1                     <pip>
	cryptography-vectors      2.3.1                 py36_1000    conda-forge
	curl                      7.63.0               h74213dd_0    conda-forge
	cvxopt                    1.2.3                     <pip>
	cycler                    0.10.0                     py_1    conda-forge
	cython                    0.28.5           py36hfc679d8_0    conda-forge
	cytoolz                   0.9.0.1          py36h470a237_1    conda-forge
	dask-core                 1.0.0                      py_0    conda-forge
	dbus                      1.13.0               h3a4f0e9_0    conda-forge
	decorator                 4.4.0                     <pip>
	decorator                 4.3.0                      py_0    conda-forge
	dill                      0.2.8.2               py36_1000    conda-forge
	entrypoints               0.2.3                 py36_1002    conda-forge
	expat                     2.2.5                hfc679d8_2    conda-forge
	fastcache                 1.0.2            py36h470a237_1    conda-forge
	fontconfig                2.13.1               h65d0f4c_0    conda-forge
	freetype                  2.9.1                h6debe1e_4    conda-forge
	gettext                   0.19.8.1             h5e8e0c9_1    conda-forge
	glib                      2.56.2               h464dc38_1    conda-forge
	gmp                       6.1.2                hfc679d8_0    conda-forge
	gmpy2                     2.0.8            py36hb705a9b_2    conda-forge
	graphite2                 1.3.13               hfc679d8_0    conda-forge
	gsl                       2.2.1           blas_openblashddceaf2_6  [blas_openblas]  conda-forge
	gst-plugins-base          1.12.5               hde13a9d_0    conda-forge
	gstreamer                 1.12.5               h5856ed1_0    conda-forge
	h5py                      2.7.1                    py36_3    conda-forge
	h5py                      2.9.0                     <pip>
	harfbuzz                  1.9.0                h04dbb29_1    conda-forge
	hdf5                      1.10.1                        2    conda-forge
	icu                       58.2                 hfc679d8_0    conda-forge
	idna                      2.8                   py36_1000    conda-forge
	idna                      2.8                       <pip>
	imageio                   2.4.1                 py36_1000    conda-forge
	ipykernel                 5.1.0           py36h24bf2e0_1001    conda-forge
	ipython                   7.2.0           py36h24bf2e0_1000    conda-forge
	ipython_genutils          0.2.0                      py_1    conda-forge
	ipywidgets                7.2.1                    py36_1    conda-forge
	jedi                      0.13.2                py36_1000    conda-forge
	jinja2                    2.10                       py_1    conda-forge
	jpeg                      9c                   h470a237_1    conda-forge
	jsonschema                3.0.0a3               py36_1000    conda-forge
	jsonschema                2.6.0                     <pip>
	jupyter-contrib-core      0.3.3                     <pip>
	jupyter-contrib-nbextensions 0.5.1                     <pip>
	jupyter-highlight-selected-word 0.2.0                     <pip>
	jupyter-latex-envs        1.4.6                     <pip>
	jupyter-nbextensions-configurator 0.4.1                     <pip>
	jupyter_client            5.2.4                      py_0    conda-forge
	jupyter_core              4.4.0                      py_0    conda-forge
	jupyterhub                0.9.4                 py36_1000    conda-forge
	jupyterlab                0.35.4                   py36_0    conda-forge
	jupyterlab_server         0.2.0                      py_0    conda-forge
	kiwisolver                1.0.1            py36h2d50403_2    conda-forge
	krb5                      1.16.2               hbb41f41_0    conda-forge
	libcurl                   7.63.0               hbdb9355_0    conda-forge
	libedit                   3.1.20170329         haf1bffa_1    conda-forge
	libffi                    3.2.1                hfc679d8_5    conda-forge
	libgcc                    7.2.0                h69d50b8_2    conda-forge
	libgcc-ng                 7.2.0                hdf63c60_3    conda-forge
	libgfortran               3.0.0                         1    conda-forge
	libiconv                  1.15                 h470a237_3    conda-forge
	libpng                    1.6.36               ha92aebf_0    conda-forge
	libprotobuf               3.6.1                hd28b015_0    conda-forge
	libsodium                 1.0.16               h470a237_1    conda-forge
	libssh2                   1.8.0                h5b517e9_3    conda-forge
	libstdcxx-ng              7.2.0                hdf63c60_3    conda-forge
	libtiff                   4.0.10               he6b73bb_1    conda-forge
	libuuid                   2.32.1               h470a237_2    conda-forge
	libxcb                    1.13                 h470a237_2    conda-forge
	libxml2                   2.9.8                h422b904_5    conda-forge
	llvmlite                  0.23.0                   py36_1    conda-forge
	lxml                      4.3.3                     <pip>
	make                      4.2.1             h470a237_1002    conda-forge
	mako                      1.0.7                      py_1    conda-forge
	markupsafe                1.1.0            py36h470a237_0    conda-forge
	marshmallow               2.19.1                    <pip>
	marshmallow-polyfield     3.2                       <pip>
	matplotlib                2.2.3            py36h8e2386c_0    conda-forge
	mistune                   0.8.4            py36h470a237_0    conda-forge
	mpc                       1.1.0                hb705a9b_6    conda-forge
	mpfr                      4.0.1                h16a7912_0    conda-forge
	mpmath                    1.1.0                      py_0    conda-forge
	mpmath                    1.1.0                     <pip>
	nbconvert                 5.3.1                      py_1    conda-forge
	nbformat                  4.4.0                      py_1    conda-forge
	nbgitpuller               0.6.1                     <pip>
	ncurses                   6.1                  hfc679d8_2    conda-forge
	networkx                  2.2                        py_1    conda-forge
	networkx                  2.2                       <pip>
	nodejs                    8.10.0                        0    conda-forge
	notebook                  5.7.2                 py36_1000    conda-forge
	ntlm-auth                 1.2.0                     <pip>
	numba                     0.38.1                   py36_0    conda-forge
	numexpr                   2.6.9            py36hf8a1672_0    conda-forge
	numpy                     1.16.2                    <pip>
	olefile                   0.46                       py_0    conda-forge
	openblas                  0.3.3                ha44fe06_1    conda-forge
	openssl                   1.0.2p               h470a237_1    conda-forge
	packaging                 18.0                       py_0    conda-forge
	pamela                    0.3.0                   py_1001    conda-forge
	pandas                    0.23.4           py36hf8a1672_0    conda-forge
	pandoc                    2.5                           0    conda-forge
	pandocfilters             1.4.2                      py_1    conda-forge
	pango                     1.40.14              he752989_2    conda-forge
	parso                     0.3.1                      py_0    conda-forge
	patsy                     0.5.1                      py_0    conda-forge
	pcre                      8.41                 hfc679d8_3    conda-forge
	pexpect                   4.6.0                 py36_1000    conda-forge
	pickleshare               0.7.5                 py36_1000    conda-forge
	Pillow                    5.4.1                     <pip>
	pillow                    5.3.0            py36hc736899_0    conda-forge
	pip                       18.1                  py36_1000    conda-forge
	pixman                    0.34.0               h470a237_3    conda-forge
	ply                       3.11                      <pip>
	prometheus_client         0.5.0                      py_0    conda-forge
	prompt_toolkit            2.0.7                      py_0    conda-forge
	protobuf                  3.6.1            py36hfc679d8_1    conda-forge
	psutil                    5.6.1                     <pip>
	pthread-stubs             0.4                  h470a237_1    conda-forge
	ptyprocess                0.6.0                 py36_1000    conda-forge
	pycodestyle               2.5.0                     <pip>
	pycosat                   0.6.3            py36h470a237_1    conda-forge
	pycparser                 2.19                      <pip>
	pycparser                 2.19                       py_0    conda-forge
	pycurl                    7.43.0.2         py36hb7f436b_0    defaults
	pygments                  2.3.1                      py_0    conda-forge
	pyopenssl                 18.0.0                py36_1000    conda-forge
	pyparsing                 2.3.0                      py_0    conda-forge
	pyrsistent                0.14.8           py36h470a237_0    conda-forge
	pysocks                   1.6.8                 py36_1002    conda-forge
	python                    3.6.7                h5001a0f_1    conda-forge
	python-dateutil           2.7.5                      py_0    conda-forge
	python-editor             1.0.3                      py_0    conda-forge
	python-oauth2             1.0.1                    py36_0    conda-forge
	pytz                      2018.7                     py_0    conda-forge
	pywavelets                1.0.1            py36h7eb728f_0    conda-forge
	pyyaml                    3.13             py36h470a237_1    conda-forge
	pyzmq                     17.1.2           py36hae99301_1    conda-forge
	qiskit                    0.8.0                     <pip>
	qiskit-aer                0.1.1                     <pip>
	qiskit-aqua               0.4.1                     <pip>
	qiskit-chemistry          0.4.2                     <pip>
	qiskit-ignis              0.1.0                     <pip>
	qiskit-terra              0.7.1                     <pip>
	r-assertthat              0.2.0           r351h6115d3f_1001    conda-forge
	r-backports               1.1.2            r351hc070d10_1    conda-forge
	r-base                    3.5.1                h391c2eb_5    conda-forge
	r-base64enc               0.1_3            r351hc070d10_2    conda-forge
	r-bh                      1.66.0_1              r351_2001    conda-forge
	r-bindr                   0.1.1           r351h6115d3f_1001    conda-forge
	r-bindrcpp                0.2.2            r351h9d2a408_1    conda-forge
	r-bit                     1.1_12           r351h470a237_2    conda-forge
	r-bit64                   0.9_7            r351hc070d10_0    conda-forge
	r-bitops                  1.0_6            r351hc070d10_2    conda-forge
	r-blob                    1.1.1                 r351_1001    conda-forge
	r-broom                   0.5.1           r351h6115d3f_1000    conda-forge
	r-callr                   3.0.0           r351h6115d3f_1000    conda-forge
	r-caret                   6.0_81           r351hc070d10_0    conda-forge
	r-cellranger              1.1.0           r351h6115d3f_1001    conda-forge
	r-class                   7.3_14           r351hc070d10_2    conda-forge
	r-cli                     1.0.1           r351h6115d3f_1000    conda-forge
	r-clipr                   0.4.1           r351h6115d3f_1001    conda-forge
	r-codetools               0.2_15          r351h6115d3f_1001    conda-forge
	r-colorspace              1.3_2            r351hc070d10_2    conda-forge
	r-config                  0.3             r351h6115d3f_1001    conda-forge
	r-crayon                  1.3.4           r351h6115d3f_1001    conda-forge
	r-curl                    3.2              r351hc070d10_2    conda-forge
	r-dbi                     1.0.0           r351h6115d3f_1001    conda-forge
	r-dbplyr                  1.2.2           r351h6115d3f_1001    conda-forge
	r-devtools                1.13.6           r351h6115d3f_1    conda-forge
	r-digest                  0.6.18           r351hc070d10_0    conda-forge
	r-dplyr                   0.7.8            r351h9d2a408_0    conda-forge
	r-evaluate                0.12            r351h6115d3f_1000    conda-forge
	r-fansi                   0.3.0            r351hc070d10_0    conda-forge
	r-forcats                 0.3.0           r351h6115d3f_1001    conda-forge
	r-foreach                 1.4.4           r351h6115d3f_1001    conda-forge
	r-forecast                8.2              r351hfc679d8_1    conda-forge
	r-forge                   0.1.0           r351h6115d3f_1000    conda-forge
	r-fracdiff                1.4_2            r351hc070d10_2    conda-forge
	r-generics                0.0.2           r351h6115d3f_1001    conda-forge
	r-ggplot2                 3.1.0           r351h6115d3f_1000    conda-forge
	r-git2r                   0.23.0           r351h1b3b946_0    conda-forge
	r-glue                    1.3.0            r351h470a237_2    conda-forge
	r-gower                   0.1.2            r351hc070d10_2    conda-forge
	r-gtable                  0.2.0           r351h6115d3f_1001    conda-forge
	r-haven                   2.0.0            r351h9d2a408_0    conda-forge
	r-hexbin                  1.27.2           r351h364d78e_2    conda-forge
	r-highr                   0.7             r351h6115d3f_1001    conda-forge
	r-hms                     0.4.2           r351h6115d3f_1000    conda-forge
	r-htmltools               0.3.6            r351hfc679d8_2    conda-forge
	r-htmlwidgets             1.2             r351h6115d3f_1000    conda-forge
	r-httpuv                  1.4.5.1          r351hfc679d8_0    conda-forge
	r-httr                    1.4.0           r351h6115d3f_1000    conda-forge
	r-ipred                   0.9_7            r351hc070d10_1    conda-forge
	r-irdisplay               0.7                   r351_1000    conda-forge
	r-irkernel                0.8.14          r351h6115d3f_1000    conda-forge
	r-iterators               1.0.10          r351h6115d3f_1001    conda-forge
	r-jsonlite                1.6              r351hc070d10_0    conda-forge
	r-kernsmooth              2.23_15          r351h364d78e_2    conda-forge
	r-knitr                   1.21            r351h6115d3f_1000    conda-forge
	r-labeling                0.3             r351h6115d3f_1001    conda-forge
	r-later                   0.7.3            r351h9d2a408_0    conda-forge
	r-lattice                 0.20_38          r351hc070d10_0    conda-forge
	r-lava                    1.6.3           r351h6115d3f_1001    conda-forge
	r-lazyeval                0.2.1            r351hc070d10_2    conda-forge
	r-lmtest                  0.9_36           r351h364d78e_0    conda-forge
	r-lubridate               1.7.4            r351h9d2a408_1    conda-forge
	r-magrittr                1.5             r351h6115d3f_1001    conda-forge
	r-markdown                0.8              r351hc070d10_3    conda-forge
	r-mass                    7.3_50           r351hc070d10_2    conda-forge
	r-matrix                  1.2_14           r351hc070d10_2    conda-forge
	r-memoise                 1.1.0           r351h6115d3f_1001    conda-forge
	r-mgcv                    1.8_24           r351hc070d10_2    conda-forge
	r-mime                    0.5              r351hc070d10_2    conda-forge
	r-modelmetrics            1.1.0            r351h9d2a408_2    conda-forge
	r-modelr                  0.1.2           r351h6115d3f_1001    conda-forge
	r-munsell                 0.5.0           r351h6115d3f_1001    conda-forge
	r-nlme                    3.1_137          r351h364d78e_0    conda-forge
	r-nnet                    7.3_12           r351hc070d10_2    conda-forge
	r-numderiv                2016.8_1        r351h6115d3f_1001    conda-forge
	r-nycflights13            1.0.0                 r351_1000    conda-forge
	r-openssl                 1.0.2            r351h9f97512_0    conda-forge
	r-pbdzmq                  0.3_3            r351hc862886_0    conda-forge
	r-pillar                  1.3.0           r351h6115d3f_1000    conda-forge
	r-pkgconfig               2.0.2           r351h6115d3f_1001    conda-forge
	r-plogr                   0.2.0           r351h6115d3f_1001    conda-forge
	r-plyr                    1.8.4            r351h9d2a408_2    conda-forge
	r-prettyunits             1.0.2           r351h6115d3f_1001    conda-forge
	r-processx                3.2.0            r351hc070d10_1    conda-forge
	r-prodlim                 2018.04.18       r351h9d2a408_2    conda-forge
	r-progress                1.2.0           r351h6115d3f_1002    conda-forge
	r-promises                1.0.1            r351h9d2a408_0    conda-forge
	r-ps                      1.1.0            r351hc070d10_1    conda-forge
	r-purrr                   0.2.5            r351hc070d10_2    conda-forge
	r-quadprog                1.5_5            r351h364d78e_2    conda-forge
	r-quantmod                0.4_13          r351h6115d3f_1000    conda-forge
	r-r2d3                    0.2.2           r351h6115d3f_1000    conda-forge
	r-r6                      2.2.2           r351h6115d3f_1001    conda-forge
	r-randomforest            4.6_14           r351h364d78e_0    conda-forge
	r-rappdirs                0.3.1            r351hc070d10_2    conda-forge
	r-rcolorbrewer            1.1_2           r351h6115d3f_1001    conda-forge
	r-rcpp                    1.0.0            r351h9d2a408_0    conda-forge
	r-rcpparmadillo           0.9.100.5.0      r351h9d2a408_0    conda-forge
	r-rcpproll                0.3.0            r351h9d2a408_0    conda-forge
	r-rcurl                   1.95_4.11        r351hc070d10_3    conda-forge
	r-readr                   1.3.1            r351h9d2a408_0    conda-forge
	r-readxl                  1.2.0            r351h9d2a408_0    conda-forge
	r-recipes                 0.1.4           r351h6115d3f_1000    conda-forge
	r-rematch                 1.0.1           r351h6115d3f_1001    conda-forge
	r-repr                    0.19                  r351_1000    conda-forge
	r-reprex                  0.2.0           r351h6115d3f_1001    conda-forge
	r-reshape2                1.4.3            r351h9d2a408_2    conda-forge
	r-rlang                   0.3.0.1          r351h470a237_0    conda-forge
	r-rmarkdown               1.11            r351h6115d3f_1000    conda-forge
	r-rpart                   4.1_13           r351hc070d10_2    conda-forge
	r-rprojroot               1.3_2           r351h6115d3f_1001    conda-forge
	r-rsqlite                 2.1.1            r351h9d2a408_0    conda-forge
	r-rstudioapi              0.8             r351h6115d3f_1001    conda-forge
	r-rvest                   0.3.2           r351h6115d3f_1001    conda-forge
	r-scales                  1.0.0            r351h9d2a408_1    conda-forge
	r-selectr                 0.4_1           r351h6115d3f_1000    conda-forge
	r-shiny                   1.2.0                 r351_1000    conda-forge
	r-sourcetools             0.1.7            r351hfc679d8_0    conda-forge
	r-sparklyr                0.9.3           r351h6115d3f_1000    conda-forge
	r-squarem                 2017.10_1       r351h6115d3f_1001    conda-forge
	r-stringi                 1.2.4            r351h9d2a408_1    conda-forge
	r-stringr                 1.3.1           r351h6115d3f_1001    conda-forge
	r-survival                2.42_6           r351hc070d10_1    conda-forge
	r-tibble                  1.4.2            r351hc070d10_2    conda-forge
	r-tidyr                   0.8.2            r351h9d2a408_2    conda-forge
	r-tidyselect              0.2.5            r351h9d2a408_0    conda-forge
	r-tidyverse               1.2.1           r351h6115d3f_1001    conda-forge
	r-timedate                3043.102        r351h6115d3f_1000    conda-forge
	r-tinytex                 0.9             r351h6115d3f_1000    conda-forge
	r-tseries                 0.10_46          r351ha44fe06_0    conda-forge
	r-ttr                     0.23_4           r351h364d78e_0    conda-forge
	r-utf8                    1.1.4            r351hc070d10_0    conda-forge
	r-uuid                    0.1_2            r351hc070d10_1    conda-forge
	r-viridislite             0.3.0           r351h6115d3f_1001    conda-forge
	r-whisker                 0.3_2           r351h6115d3f_1001    conda-forge
	r-withr                   2.1.2           r351h6115d3f_1000    conda-forge
	r-xfun                    0.3             r351h6115d3f_1001    conda-forge
	r-xml2                    1.2.0            r351h9d2a408_2    conda-forge
	r-xtable                  1.8_3                 r351_2000    conda-forge
	r-xts                     0.11_1           r351hc070d10_0    conda-forge
	r-yaml                    2.2.0            r351hc070d10_1    conda-forge
	r-zoo                     1.8_4            r351hc070d10_0    conda-forge
	readline                  7.0                  haf1bffa_1    conda-forge
	requests                  2.21.0                py36_1000    conda-forge
	requests-ntlm             1.1.0                     <pip>
	rpy2                      2.9.4           py36r351h941a26a_1    conda-forge
	ruamel_yaml               0.15.71          py36h470a237_0    conda-forge
	scikit-image              0.14.1           py36hfc679d8_1    conda-forge
	scikit-learn              0.20.0                    <pip>
	scikit-learn              0.20.2          py36_blas_openblash00c3548_400  [blas_openblas]  conda-forge
	scipy                     1.1.0           py36_blas_openblashb06ca3d_202  [blas_openblas]  conda-forge
	scipy                     1.2.1                     <pip>
	seaborn                   0.9.0                      py_0    conda-forge
	send2trash                1.5.0                      py_0    conda-forge
	setuptools                40.6.3                   py36_0    conda-forge
	setuptools                40.8.0                    <pip>
	sip                       4.18.1           py36hfc679d8_0    conda-forge
	six                       1.12.0                    <pip>
	six                       1.12.0                py36_1000    conda-forge
	sqlalchemy                1.2.15           py36h470a237_0    conda-forge
	sqlite                    3.26.0               hb1c47c0_0    conda-forge
	statsmodels               0.9.0            py36h7eb728f_0    conda-forge
	sympy                     1.1.1                    py36_0    conda-forge
	sympy                     1.3                       <pip>
	terminado                 0.8.1                 py36_1001    conda-forge
	testpath                  0.4.2                 py36_1000    conda-forge
	tini                      0.18.0               h470a237_1    conda-forge
	tk                        8.6.9                ha92aebf_0    conda-forge
	tktable                   2.10                 h14c3975_0    defaults
	toolz                     0.9.0                      py_1    conda-forge
	tornado                   5.1.1            py36h470a237_0    conda-forge
	traitlets                 4.3.2                 py36_1000    conda-forge
	tzlocal                   1.5.1                      py_0    conda-forge
	urllib3                   1.24.1                py36_1000    conda-forge
	urllib3                   1.24.1                    <pip>
	vincent                   0.4.4                      py_1    conda-forge
	wcwidth                   0.1.7                      py_1    conda-forge
	webencodings              0.5.1                      py_1    conda-forge
	wheel                     0.32.3                   py36_0    conda-forge
	widgetsnbextension        3.2.1                    py36_1    conda-forge
	xlrd                      1.2.0                      py_0    conda-forge
	xorg-kbproto              1.0.7                h470a237_2    conda-forge
	xorg-libice               1.0.9                h470a237_4    conda-forge
	xorg-libsm                1.2.3                h8c8a85c_0    conda-forge
	xorg-libx11               1.6.6                h470a237_0    conda-forge
	xorg-libxau               1.0.8                h470a237_6    conda-forge
	xorg-libxdmcp             1.1.2                h470a237_7    conda-forge
	xorg-libxext              1.3.3                h470a237_4    conda-forge
	xorg-libxrender           0.9.10               h470a237_2    conda-forge
	xorg-renderproto          0.11.1               h470a237_2    conda-forge
	xorg-xextproto            7.3.0                h470a237_2    conda-forge
	xorg-xproto               7.0.31               h470a237_7    conda-forge
	xz                        5.2.4                h470a237_1    conda-forge
	yaml                      0.1.7                h470a237_1    conda-forge
	zeromq                    4.2.5                hfc679d8_6    conda-forge
	zlib                      1.2.11               h470a237_3    conda-forge
```

I found some related recent errors like https://github.com/jupyterhub/jupyter-server-proxy/issues/109

Thanks

**Comment by minrk** [comment]
Indeed, we should be wrapping it in `maybe_future`. This should come up rarely, because it only happens for incorrect URLs (i.e. every 500 error caused by this *should* be a 404). In this case, I think it's `/hub/api/users` not `/hub/api/users/` (trailing slash is incorrect).

**Comment by minrk** [comment]
This is also fixed in master, which will be released as 1.0 before long, so I think we can close.

**Comment by cbjuan** [comment]
Thanks for the clarification about 500 errors. Agree on closing the issue
